modules/components/pages/processors/aws_bedrock_embeddings.adoc

```diff
@@ -0,0 +1,203 @@
+= aws_bedrock_embeddings
+// tag::single-source[]
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Generates vector embeddings from text prompts, using the https://aws.amazon.com/bedrock/[AWS Bedrock API^].
+
+ifndef::env-cloud[]
+Introduced in version 4.37.0.
+endif::[]
+
+ifndef::env-cloud[]
+[tabs]
+======
+Common::
++
+--
+```yml
+# Common config fields, showing default values
+label: ""
+aws_bedrock_embeddings:
+  model: amazon.titan-embed-text-v1 # No default (required)
+  text: "" # No default (optional)
+```
+--
+Advanced::
++
+--
+```yml
+# All config fields, showing default values
+label: ""
+aws_bedrock_embeddings:
+  region: ""
+  endpoint: ""
+  credentials:
+    profile: ""
+    id: ""
+    secret: ""
+    token: ""
+    from_ec2_role: false
+    role: ""
+    role_external_id: ""
+  model: amazon.titan-embed-text-v1 # No default (required)
+  text: "" # No default (optional)
+```
+--
+======
+endif::[]
+
+ifdef::env-cloud[]
+[tabs]
+======
+Common::
++
+--
+```yml
+# Common config fields, showing default values
+label: ""
+aws_bedrock_embeddings:
+  model: amazon.titan-embed-text-v1 # No default (required)
+  text: "" # No default (optional)
+```
+--
+Advanced::
++
+--
+```yml
+# All config fields, showing default values
+label: ""
+aws_bedrock_embeddings:
+  region: ""
+  endpoint: ""
+  credentials:
+    from_ec2_role: false
+    role: ""
+    role_external_id: ""
+  model: amazon.titan-embed-text-v1 # No default (required)
+  text: "" # No default (optional)
+```
+--
+======
+endif::[]
+
+This processor sends text prompts to your chosen large language model (LLM) which generates vector embeddings for them, using the AWS Bedrock API.
+
+For more information, see the https://docs.aws.amazon.com/bedrock/latest/userguide[AWS Bedrock documentation^].
+
+== Fields
+
+=== `model`
+
+The ID of the LLM that you want to use to generate vector embeddings. For a full list, see the https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html[AWS Bedrock documentation^].
+
+*Type*: `string`
+```yml
+# Examples
+model: amazon.titan-embed-text-v1
+model: amazon.titan-embed-text-v2:0
+model: cohere.embed-english-v3
+model: cohere.embed-multilingual-v3
+```
+=== `text`
+
+The prompt you want to generate a vector embedding for. By default, the processor submits the entire payload as a string.
```

Comment:
```suggestion
The prompt you want to generate a vector embedding for. The processor submits the entire payload as a string.
```

If it's not configurable, there's no need to mention that it's the default.

Reply:
Updated as requested.